apps/fishing-map/next.config.js

```diff
@@ -25,7 +25,7 @@ const nextConfig = {
   webpack: function (config) {
     config.resolve.alias = {
       ...config.resolve.alias,
-      'mapbox-gl': '@globalfishingwatch/mapbox-gl',
+      'mapbox-gl': '@globalfishingwatch/maplibre-gl',
```

Comment:
yes we do because this allows us to use the react-map-gl library without our fork, see [this example](https://github.com/visgl/react-map-gl/blob/master/examples/maplibre/webpack.config.js#L25) of using maplibre